test/test_tensorboard.py

```diff
@@ -397,6 +398,19 @@ def test_wrong_input_size(self):
                 with SummaryWriter(comment='expect_error') as w:
                     w.add_graph(model, dummy_input)  # error
 
+        def test_torchvision_smoke(self):
+            model_name_input = {
+                'alexnet': (2, 3, 224, 224),
+                'resnet34': (2, 3, 224, 224),
+                'densenet121': (2, 3, 224, 224),
+                'vgg19': (2, 3, 224, 224),
+                # 'mobilenet_v2': (2, 3, 224, 224),  # will fail onnx
```

Comment:
Why does this fail onnx?

Reply:
Also, any interest in adding `vgg16` and `restnet50` to this list?

Reply:
Should be: fail `_optimize_graph`. let me check the root cause tmr.

Reply:
https://github.com/pytorch/vision/blob/dc7db17e7724b9d56cbd45931b0788540aaf2586/torchvision/models/mobilenet.py#L97
Changing this line to two consecutive calls to `x = x.mean(dim=2)` can fix.

Reply:
@suo and @houseroad, any idea why this would need to be changed? Thanks.

Reply:
error message with original code:
```
Traceback (most recent call last):
  File "test_tensorboard.py", line 426, in test_torchvision_smoke
    w.add_graph(model, torch.zeros(input_shape))
  File "/Users/dexter/git/pytorch/torch/utils/tensorboard/writer.py", line 534, in add_graph
    self._get_file_writer().add_graph(graph(model, input_to_model, verbose, **kwargs))
  File "/Users/dexter/git/pytorch/torch/utils/tensorboard/_pytorch_graph.py", line 298, in graph
    _optimize_trace(trace, operator_export_type)
  File "/Users/dexter/git/pytorch/torch/utils/tensorboard/_pytorch_graph.py", line 231, in _optimize_trace
    trace.set_graph(_optimize_graph(trace.graph(), operator_export_type))
  File "/Users/dexter/git/pytorch/torch/utils/tensorboard/_pytorch_graph.py", line 262, in _optimize_graph
    graph = torch._C._jit_pass_onnx(graph, operator_export_type)
  File "/Users/dexter/git/pytorch/torch/onnx/__init__.py", line 50, in _run_symbolic_function
    return utils._run_symbolic_function(*args, **kwargs)
  File "/Users/dexter/git/pytorch/torch/onnx/utils.py", line 589, in _run_symbolic_function
    return fn(g, *inputs, **attrs)
  File "/Users/dexter/git/pytorch/torch/onnx/symbolic.py", line 412, in symbolic
    dim, keepdim = _get_const(dim, 'i', 'dim'), _get_const(keepdim, 'i', 'keepdim')
  File "/Users/dexter/git/pytorch/torch/onnx/symbolic.py", line 116, in _get_const
    return _parse_arg(value, desc)
  File "/Users/dexter/git/pytorch/torch/onnx/symbolic.py", line 75, in _parse_arg
    return int(tval)
ValueError: only one element tensors can be converted to Python scalars
```